src/loading2/octree-loader.ts

```diff
@@ -313,6 +322,12 @@ const typenameTypeattributeMap = {
 
 type AttributeType = keyof typeof typenameTypeattributeMap;
 
+type BufferView = {
+	byteLength: number,
+	byteOffset: number,
```

Comment:
I don't see these 2 properties being used anywhere. Could you explain why this is needed? What am I missing?

Reply:
These are needed when using combined attribute buffers (multiple attributes in a single file). In order not to bloat the scope of this PR too much I will open another one with the changes for this particular feature. However, since we know they will be written into the metadata I have added them here already.

Reply:
👍 let's keep it then. (Ideally though, it should've been added with this other PR)